PocketKit/Sources/PocketKit/MyList/SortFilterMenu/SortMenuViewModel.swift

```diff
@@ -0,0 +1,41 @@
+import UIKit
+import Sync
+import Analytics
+
+
+class SortMenuViewModel {
+
+    typealias Snapshot = NSDiffableDataSourceSnapshot<SortSection, SortRow>
+    @Published
+    private var _snapshot = Snapshot()
+    var snapshot: Published<Snapshot>.Publisher { $_snapshot }
+
+    private let source: Source
+    private let tracker: Tracker
+    private let listOptions: ListOptions
+
+    init(source: Source, tracker: Tracker, listOptions: ListOptions) {
+        self.source = source
+        self.tracker = tracker
+        self.listOptions = listOptions
+        configureDiffable()
+    }
+
+    func configureDiffable() {
+        var snapshotTemp: NSDiffableDataSourceSnapshot<SortSection, SortRow> = .init()
+        snapshotTemp.appendSections([.sortBy])
+        snapshotTemp.appendItems([.newest, .oldest], toSection: .sortBy)
+        _snapshot = snapshotTemp
+    }
+    
+    func cellViewModel(for row: SortRow) -> SortMenuViewCell.Model {
+        return .init(attributedTitle: NSAttributedString(string: row.rawValue, style: .header.sansSerif.h8), isSelected: (listOptions.selectedSort.rawValue == row.rawValue) ? true : false)
```

Comment:
for these longer function calls, we prefer to chop them down to make them a bit easier on the eyes.

e.g.:
```swift
.init(
    attributedTitle: NSAttributedString(string: row.rawValue, style: .header.sansSerif.h8),
    isSelected: (listOptions.selectedSort.rawValue == row.rawValue) ? true : false
)
```

Also I don't think you need the ternary operator in this case. Can you simplify to this?
```swift
isSelected: lisOptions.selectedSort == row
```

Reply:
thank you